src/bindings/system/fs.rs

```diff
@@ -190,6 +192,25 @@ pub fn init(lua: &Lua) -> ::Result<()> {
             .map_err(LuaError::external)
     })?)?;
 
+	module.set("copy_file", lua.create_function(|_, (src, dest): (String, String)| {
+		fs::copy(src, dest)
+			.map_err(LuaError::external)
+	})?)?;
+
+	// TODO custom implementation
+	module.set("copy_dir", lua.create_function(|_, (src, dest): (String, String)| {
```

Comment:
For a custom implementation, one could iterate throughout the directory and creating directories that dont exist in the `dest` that exist in `src` while using `fs::copy` to copy files. Should be pretty easy to setup.